application/event_processor/change_event_validation.py

```diff
@@ -55,27 +58,28 @@ def check_ods_code_length(odscode: str) -> None:
     Args:
         odscode (str): odscode of NHS UK service
     """
+    expected_length = 5
```

Comment:
Is odscode  length is minimum of 5 and exactly 5

Reply:
Yes. Expected odscodes from NHSUK payloads should not exceed 5 characters.